src/ResourceManager/ContainerInstance/Commands.ContainerInstance.Test/ScenarioTests/ContainerInstanceTests.cs

```diff
@@ -60,7 +74,31 @@ public void TestContainerInstanceLog()
         [Trait(Category.AcceptanceType, Category.CheckIn)]
         public void TestCreateContainerGroupWithVolume()
         {
-            TestController.NewInstance.RunPowerShellTest(_logger, "Test-AzureRmContainerGroupWithVolumeMount");
+            TestController.NewInstance.RunPowerShellTest(_logger, "Test-AzureRmContainerGroupWithVolume");
+        }
+
+#if NETSTANDARD
```

Comment:
@allan-xiazhan we can remove these ifdefs, right? The `-Command` parameter is enabled in .NET Standard as of this PR

Reply:
I believe so. Good catch!